deployer/src/config_creator.py

```diff
@@ -74,23 +74,90 @@ def to_gitbook_config(config):
 
 def to_pkgdown_config(config, urls=None):
     if urls:
-        config["sitemap_urls"] = [
-            extract_root_from_input(urls[0]) + "sitemap.xml"]
+        root = extract_root_from_input(urls[0])
+        config["start_urls"] = [{
+            "url": root + "index.html",
+            "selectors_key": "homepage",
+            "tags": [
+                "homepage"
+            ]
+        },
+            {
+                "url": root + "reference",
+                "selectors_key": "reference",
+                "tags": [
+                    "reference"
+                ]
+            },
+            {
+                "url": root + "articles",
+                "selectors_key": "articles",
+                "tags": [
+                    "articles"
+                ]
+            }]
 
-    config["selectors"]["lvl0"] = OrderedDict((
-        ("selector", ".contents h1"),
-        ("default_value", "Documentation")
+        config["sitemap_urls"] = [
+            root + "sitemap.xml"]
+
+    config["selectors"] = OrderedDict((
+        ("homepage", OrderedDict((
+            ("lvl0", OrderedDict((
+                ("selector", ".contents h1"),
+                ("default_value", "pkgdown Home page")
+            ))),
+            ("lvl1", ".contents h2"),
+            ("lvl2", OrderedDict((
+                ("selector", ".contents h3"),
+                ("default_value", "Context")
+            ))),
+            ("lvl3", ".ref-arguments td, .ref-description"),
+            ("text", ".contents p, .contents li, .contents .pre")
+        ))),
+        ("reference", OrderedDict((
+            ("lvl0", ".contents h1"),
+            ("lvl1", OrderedDict((
+                ("selector", ".contents .name"),
+                ("default_value", "Argument")
+            ))),
+            ("lvl2", OrderedDict((
+                ("selector", ".ref-arguments th"),
+                ("default_value", "Description")
+            ))),
+            ("lvl3", ".ref-arguments td, .ref-description"),
+            ("text", ".contents p, .contents li")
+        ))),
+        ("articles", OrderedDict((
+            ("lvl0", ".contents h1"),
+            ("lvl1", ".contents .name"),
+            ("lvl2", OrderedDict((
+                ("selector", ".contents h2, .contents h3"),
+                ("default_value", "Context")
+            ))),
+            ("text",
+             ".contents p, .contents li, .tempate-article .contents .pre")
```

Comment:
Might be a typo: `tempate-` instead of `.template-`

Reply:
You are write. I have removed it as the corrected selector from it is not matching any element from the websites.
Fixed in configuration too https://github.com/algolia/docsearch-configs/commit/538370ba764bd45102258c00644f2a86fee49588